src/app/zap-templates/zcl/data-model/chip/web-rtc-provider-cluster.xml

```diff
@@ -0,0 +1,145 @@
+<?xml version="1.0"?>
+<!--
+Copyright (c) 2024 Project CHIP Authors
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/app_clusters/WebRTC_Provider.adoc
+Parameters: in-progress 
+Git: 0.7-fall2024-ncr-1158-g413e195c2
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
+  <domain name="General"/>
+
+  <enum name="StreamTypeEnum" type="enum8">
+    <cluster code="0x0553"/>
+    <cluster code="0x0554"/>
+    <item name="Internal" value="0x00"/>
+    <item name="Recording" value="0x01"/>
+    <item name="Analysis" value="0x02"/>
+    <item name="LiveView" value="0x03"/>
+  </enum>
+
+  <bitmap name="WebRTCMetadataOptions" type="bitmap8">
+    <cluster code="0x0553"/>
+    <cluster code="0x0554"/>
+    <field name="DataTLV" mask="0x01"/>
+  </bitmap>
+
+  <enum name="WebRTCEndReasonEnum" type="enum8">
+    <cluster code="0x0553"/>
+    <cluster code="0x0554"/>
+    <item name="ICE_FAILED" value="0x01"/>
+    <item name="ICE_TIMEOUT" value="0x02"/>
+    <item name="USER_HANGUP" value="0x03"/>
+    <item name="USER_BUSY" value="0x04"/>
+    <item name="REPLACED" value="0x05"/>
+    <item name="NO_USER_MEDIA" value="0x06"/>
+    <item name="INVITE_TIMEOUT" value="0x07"/>
+    <item name="ANSWERED_ELSEWHERE" value="0x08"/>
+    <item name="OUT_OF_RESOURCES" value="0x09"/>
+    <item name="MEDIA_TIMEOUT" value="0x0A"/>
+    <item name="LOW_POWER." value="0x0B"/>
+    <item name="UNKNOWN_ERROR" value="0x0C"/>
+  </enum>
+
+  <struct name="ICEServerStruct" apiMaturity="provisional">
+    <cluster code="0x0553"/>
+    <cluster code="0x0554"/>
+    <item fieldId="1" name="urls" array="true" type="char_string"/>
```

Comment:
Field names usually start with a capital letter.  Also, "URLs" would be the usual capitalization, I would think.  Please raise spec issues for this one and the other fields in this struct.

Reply:
Outstanding question in spec PR @ https://github.com/CHIP-Specifications/connectedhomeip-spec/pull/10004#discussion_r1783441957

Reply:
Resolving for now, can re-run alchemy after the outcome of the discussion in spec